deployment/ansible/group_vars/all.example

```diff
@@ -19,7 +19,7 @@ postgresql_port: 5432
 postgresql_version: "9.4"
 postgresql_package_version: "9.4.*.pgdg14.04+1"
 postgresql_support_repository_channel: "main"
-postgresql_support_libpq_version: "9.5.*.pgdg14.04+1"
+postgresql_support_libpq_version: "9.3.4-1"
```

Comment:
I'd feel more comfortable if we find a way to make this at least `9.4`; JSONB fields weren't introduced until Postgres 9.4, so it'd be nice if all the versions can match up. I'm not sure whether we're actually _using_ libpq though, so if it's not breaking anything for you locally we can punt and make a separate issue for it.

Reply:
These are the versions available in the services VM:

```bash
$ sudo apt-cache show libpq-dev | grep Version
Version: 10.1-1.pgdg14.04+1
Version: 9.3.20-0ubuntu0.14.04
Version: 9.3.4-1
```

It might make sense to open a separate (probably small) issue

Reply:
Yeah, we're relying on the `azavea.postgresql-support` role to install `libpq-dev`. The `azavea.postgresql` role seems to have support for installing a wide range of postgres versions, but the `azavea.postgresql-support` role doesn't seem to have been updated quite as much. If using 9.4 for the version doesn't work we'll probably need an ops issue to add support to that role.